interface/ceed-operator.c

```diff
@@ -264,6 +264,64 @@ int CeedOperatorGetActiveElemRestriction(CeedOperator op,
   return CEED_ERROR_SUCCESS;
 }
 
+/**
+  @brief Set QFunctionContext field
```

Comment:
Something more specific about how this applies to all sub-operators.
```suggestion
  @brief Set QFunctionContext field for every QFunction containing `field_name` within the provided CeedOperator
```